src/command/Menus.js

```diff
@@ -179,6 +179,19 @@ define(function (require, exports, module) {
     }
 
     /**
+    * Removes the attached event listeners from the corresponding object.
+    * @param {Command} command
+    */
+    function removeMenuItemEventListeners(command) {
+        command
+            .off("enabledStateChange", command._enabledChanged)
```

Comment:
looking at the code now, `_enabledChanged` and other methods live on `MenuItem` not on `command` ... are you sure that you're not calling `.off("enabledStateChange", undefined)` ???